core/src/main/scala/astraea/spark/rasterframes/stats/CellHistogram.scala

```diff
@@ -49,6 +51,98 @@ case class CellHistogram(stats: CellStatistics, bins: Seq[CellHistogram.Bin]) {
 
     lines.mkString("\n")
   }
+
+  // find the count of the bin that the value fits into
+  def itemCount(label: Double): Long = {
+    // look at each pair of consecutive bins, and when one bin is <= the value
+    // and the other is > the value, return the smaller bin label
+    val sorted = bins.sortBy(_.value)
+    require(sorted.nonEmpty, "Sorted must be nonempty")
```

Comment:
Move this to the top of the method, and test against `bins` instead of `sorted`.